docs/photoniq/prerendering/prerendering-settings.md

```diff
@@ -78,9 +81,15 @@ You can block stylesheets to speed up load times.
 
 ![Stylesheet Blocking](/img/prerendering/stylesheet-blocking.png)
 
+### Inject Styled Components CSS
+
+You can inject styled components into the head element to speed up load times.
```

Comment:
Actually, this slows down the page rendering but improves styles when you're testing. @tushar-macrometa. Any thoughts?

Reply:
Updated